### PR TITLE
fix(TDS-3104/List): toolbar dropdown value should be in capitalize

### PR DESCRIPTION
**Pull request by jlame**
**What is the problem this PR is trying to solve?**
The sort by are displayed in uppercase
**What is the chosen solution to this problem?**
Display them in capitalize
**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/List/Toolbar/Toolbar.scss [hunk 1]
```diff
@@ -26,6 +26,9 @@ $tc-list-toolbar-background-color: $concrete !default;
 	:global(.navbar-nav) {
 		li a {
 			color: $black;
+			&:global(.dropdown-toggle) {
+				text-transform: capitalize;
+			}
```
Comment (talend-glorieux) [inline packages/components/src/List/Toolbar/Toolbar.scss]: Why did you need to add a global `text-transform`?

Comment (jlame) [reply packages/components/src/List/Toolbar/Toolbar.scss]: if i'm correct its because the top class is scoped locally, could have wrote it that way though : 

:global {
		.navbar-nav li a {
			color: $black;
			&.dropdown-toggle {
				text-transform: capitalize;
			}
		}
	}

which might be nicer, what do you think ?

**Comment by jlame** [comment]
@jare-talend its already part of the guideline at least in the screenshots
